server.js

```diff
@@ -1,30 +1,38 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import topMusicData from './data/top-music.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello, welcome to an API about popular Spotify tracks.')
+})
+
+// Endpoint 1, all the data with 50 popular songs on Spotify 
+app.get('/songs', (req, res) => {
+  res.json(topMusicData)
+})
+
+// Endpoint 2, sorting the data on music genre
+app.get('/genre/:genre', (req, res) => {
+  const genre = req.params.genre
+  const songGenre = topMusicData.filter((item) => item.genre === genre)
+  res.json(songGenre)
+})
```

Comment:
Wouldn't it be nice if this was a query under the `/songs` endpoint? 🤩 So that the user can write:
`/songs?genre=pop` and so on. Endpoints should be named after what they are returning, not how the data is filtered or sorted